### PR TITLE
Simple-case entry point function

### PR DESCRIPTION
**Pull request by fsoikin**
Finding a DOM element by ID and mounting an Elmish component onto it appears to be a common scenario, so we now provide an app for that.

### CODE REVIEW

**Comment by gasi** [comment]
LOL, built the same thing at the same time 😜 https://github.com/collegevine/purescript-elmish/pull/6

**Comment by fsoikin** [comment]
Yeah, but mine's better! 😛

File: src/Elmish/Boot.purs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    -> Effect Unit
+boot { domElementId, def } =
+    boot' { domElementId, def, onViewError: Console.error } >>= case _ of
+        BootOk ->
+            pure unit
+        ElementNotFound e ->
+            Console.error $ "Element #" <> e.domElementId <> " not found"
+
+-- | Mounts the given UI component to a DOM element with given ID
+boot' :: forall msg state.
```
Comment (sassela) [inline src/Elmish/Boot.purs]: Could this, or `boot`, be given a more descriptive name? eg. `bootOrError`?

Comment (fsoikin) [reply src/Elmish/Boot.purs]: Well, this specific name I did consider, but it's not really "or error", right? It doesn't actually throw errors.

I'm not against renaming in principle though, just couldn't come up with a better name.

**Pull request merged**